staging/src/k8s.io/kubectl/pkg/cmd/debug/debug.go

```diff
@@ -51,12 +51,28 @@ import (
 )
 
 var (
-	debugLong = templates.LongDesc(i18n.T(`Tools for debugging Kubernetes resources`))
+	debugLong = templates.LongDesc(i18n.T(`
+		Debug cluster resources using interactive debugging containers.
+
+		'debug' provides automation for common debugging tasks for cluster objects identified by
+		resource and name. Pods will be used by default if resource is not specified.
+
+		The action taken by 'debug' varies depending on what resource is specified. Supported
+		actions include:
+
+		* Pod: Create a copy of an existing pod with certain attributes changed, for example
```

Comment:
I'd call it workload, since it's not a problem to get a pod from a deployment/daemon set/job, etc.